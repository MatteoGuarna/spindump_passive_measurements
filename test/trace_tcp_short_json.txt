Short wget TCP trace, no security. Using JSON output.
