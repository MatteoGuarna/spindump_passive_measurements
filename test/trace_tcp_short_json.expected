[
{ "Event": "new", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59105:80", "Ts": 1553418212984046, "State": "Starting", "Packets1": 1, "Packets2": 0, "Bytes1": 84, "Bytes2": 0, "Bandwidth1": 84, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59105:80", "Ts": 1553418213017044, "State": "Up", "Right_rtt": 32998, "Packets1": 1, "Packets2": 1, "Bytes1": 84, "Bytes2": 80, "Bandwidth1": 84, "Bandwidth2": 80 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59105:80", "Ts": 1553418213017134, "State": "Up", "Left_rtt": 90, "Packets1": 1, "Packets2": 1, "Bytes1": 84, "Bytes2": 80, "Bandwidth1": 84, "Bandwidth2": 80 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59105:80", "Ts": 1553418213061925, "State": "Up", "Right_rtt": 44683, "Packets1": 3, "Packets2": 1, "Bytes1": 368, "Bytes2": 80, "Bandwidth1": 368, "Bandwidth2": 80 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59105:80", "Ts": 1553418213061986, "State": "Closing", "Left_rtt": 58, "Packets1": 3, "Packets2": 4, "Bytes1": 368, "Bytes2": 734, "Bandwidth1": 368, "Bandwidth2": 734 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59105:80", "Ts": 1553418213061987, "State": "Closing", "Left_rtt": 59, "Packets1": 4, "Packets2": 4, "Bytes1": 440, "Bytes2": 734, "Bandwidth1": 440, "Bandwidth2": 734 },
{ "Event": "new", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59106:80", "Ts": 1553418213064120, "State": "Starting", "Packets1": 1, "Packets2": 0, "Bytes1": 84, "Bytes2": 0, "Bandwidth1": 84, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59105:80", "Ts": 1553418213098429, "State": "Closed", "Right_rtt": 34370, "Packets1": 6, "Packets2": 4, "Bytes1": 584, "Bytes2": 734, "Bandwidth1": 584, "Bandwidth2": 734 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59106:80", "Ts": 1553418213098432, "State": "Up", "Right_rtt": 34312, "Packets1": 1, "Packets2": 1, "Bytes1": 84, "Bytes2": 80, "Bandwidth1": 84, "Bandwidth2": 80 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59106:80", "Ts": 1553418213098549, "State": "Up", "Left_rtt": 117, "Packets1": 1, "Packets2": 1, "Bytes1": 84, "Bytes2": 80, "Bandwidth1": 84, "Bandwidth2": 80 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59106:80", "Ts": 1553418213134370, "State": "Up", "Right_rtt": 35675, "Packets1": 3, "Packets2": 1, "Bytes1": 378, "Bytes2": 80, "Bandwidth1": 378, "Bandwidth2": 80 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59106:80", "Ts": 1553418213134415, "State": "Up", "Left_rtt": 43, "Packets1": 3, "Packets2": 4, "Bytes1": 378, "Bytes2": 3152, "Bandwidth1": 378, "Bandwidth2": 3152 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59106:80", "Ts": 1553418213134659, "State": "Up", "Left_rtt": 23, "Packets1": 4, "Packets2": 7, "Bytes1": 450, "Bytes2": 7652, "Bandwidth1": 450, "Bandwidth2": 7652 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59106:80", "Ts": 1553418213134805, "State": "Closing", "Left_rtt": 28, "Packets1": 5, "Packets2": 9, "Bytes1": 522, "Bytes2": 8254, "Bandwidth1": 522, "Bandwidth2": 8254 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59106:80", "Ts": 1553418213134806, "State": "Closing", "Left_rtt": 27, "Packets1": 6, "Packets2": 9, "Bytes1": 594, "Bytes2": 8254, "Bandwidth1": 594, "Bandwidth2": 8254 },
{ "Event": "measurement", "Type": "TCP", "Addrs": ["2001:67c:1232:144:9498:6df6:f450:110b","2001:67c:2b0:1c1::198"], "Session": "59106:80", "Ts": 1553418213169945, "State": "Closed", "Right_rtt": 33074, "Packets1": 9, "Packets2": 9, "Bytes1": 810, "Bytes2": 8254, "Bandwidth1": 810, "Bandwidth2": 8254 }
]
